app/components/govuk_component/cookie_banner.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= tag.div(class: classes, **html_attributes) do %>
```

Comment:
I think we're missing the `role="region"` and `aria-label` attributes here? See [Nunjucks version](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/cookie-banner/template.njk#L3).